### DISCUSSION TITLE
Provide a menu bar for when CSDs are disabled on Linux (gtk-titlebar=false)

### DISCUSSION THREAD

**Discussion by iamgreaser** [ghostty-org/ghostty#3154] [Feature Requests, Ideas] [answered]
Client-side decorations (CSDs) annoy me so I try to turn them off where practical. Thank you for providing a config option to do exactly this.

The downside is that I need to memorise the shortcuts that are provided by the buttons that would be placed into the title bar. I'd probably end up learning this eventually, but it is nice to have a menu one can peruse, especially for things I might only use once in a blue moon.

Judging by some other screenshots here, it does seem that on macOS there is a usable menu bar. If this is correct, then perhaps this could be a good starting point?

**Comment by jcollie** [comment]
There is a right-click context menu. It doesn't have all of the same options as the normal menu, but it could be extended if needed.

**Comment by 00-kat** [comment, marked as answer]
Duplicate of #3434.  Technically this discussion was first, but all the discussion is over there.
